task_8/task_8_10.py

```diff
@@ -0,0 +1,11 @@
+def chess():
+    coord = input('Input figure coordinates (example: d4): ')
+    if 104 < ord(coord[0]) or 57 > ord(coord[0])\
+            or 57 < ord(coord[1]) or 49 > ord(coord[1]):
+        return 'Out of chess field'
```

Comment:
rewrite to have sth < var < sth

Reply:
if 104 < ord(coord[0]) < 57 \
         or 57 < ord(coord[1]) < 49:

this construction doesn't work. Or I misunderstood?